internal/slicer/slicer.go

```diff
@@ -70,19 +71,43 @@ func Run(options *RunOptions) error {
 		targetDirAbs = filepath.Join(dir, targetDir)
 	}
 
+	orderedArchives := make([]archive.Archive, 0, len(options.Archives))
```

Comment:
this might deserve a comment explaining that the default priority for all archives is 0 and the higher the number the higher the priority (opposite of "defcon" numbering)